### ISSUE TITLE
Validate consistent TS version across finesse-frontend

### ISSUE THREAD

**Issue by redat97** [ai-cfia/github-workflows#65]
open an issue to automate validation that versions is provided consistently in our typescript projects and that the version is incremented between PR.

_Originally posted by @rngadam in https://github.com/ai-cfia/finesse-frontend/pull/45#discussion_r1404663774_

A mistake happened where the TS version was decreased so we want a check to make sure that we have consistency across the whole of finesse-frontend

**Comment by SonOfLope** [comment]
@rngadam @ThomasCardin I want to pick up this issue. Are the specs still valid ? Do we want this to be placed in the reusable workflow repo or should it stay here (that way it could be used for future projects)? I will branch from master instead of picking this branch but just want to be sure it is still needed. 

Also, I suggest to do the validation directly in the workflow script instead of using a separate JavaScript file to reduce the number of files and points of failure. 

TLDR : 
- Should I move the validation workflow to github-workflows repo ? 
- Can I implement the solution directly in the workflow file without using a js file ?
